examples/bri-3/prisma/schema.prisma

```diff
@@ -16,4 +16,36 @@ model BpiSubject {
   description String  
   type Int
   publicKey String 
+}
+
+model Order {
```

Comment:
You can remove all of these for now. They will be added in interations as each component is introduced to the codebase